packages/fiber/src/core/reconciler.tsx

```diff
@@ -19,6 +19,10 @@ import type { RootStore } from './store'
 import { removeInteractivity, type EventHandlers } from './events'
 import type { ThreeElement } from '../three-types'
 
+const DiscreteEventPriority = 2
+const ContinuousEventPriority = 8
+const DefaultEventPriority = 32
```

Comment:
Vite isn't able to parse these into static exports. Instead, it outputs a function whose keys are mapped to these constants. So I have to hardcode them here. This is because they can possibly change between development and production. We need to figure out how to handle the modality here when bundling. Both versions are bundled in the reconciler.